docs/getting-started/server/database/ef/index.mdx

```diff
@@ -196,30 +196,48 @@ If you would like to verify that everything worked correctly:
 
 ## Testing EF Changes
 
-Since we allow for multiple databases it is important that any changes to EF repositories/models are
-tested against all possible databases. You may want to use a database that is different from your
-local development database because the tests may add or remove data. To apply migrations to a
-database different from your global settings run the following commands from the root of your
-repository
-
-```bash
-# EntityFramework CLI Reference: https://learn.microsoft.com/en-us/ef/core/cli/dotnet
-
-# Migrate Postgres database ex connection string: Host=localhost;Username=postgres;Password=SET_A_PASSWORD_HERE_123;Database=vault_dev_test
-dotnet ef database update --startup-project util/PostgresMigrations --connection "[POSTGRES_CONNECTION_STRING]"
-
-# Migrate MySql database ex connection string: server=localhost;uid=root;pwd=SET_A_PASSWORD_HERE_123;database=vault_dev_test
-dotnet ef database update --startup-project util/MySqlMigrations --connection "[MYSQL_CONNECTION_STRING]"
-
-cd test/Infrastructure.IntegrationTest
-
-# https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=windows#secret-manager
-dotnet user-secrets set "Ef:Postgres" "[POSTGRES_CONNECTION_STRING]"
-dotnet user-secrets set "Ef:MySql" "[MYSQL_CONNECTION_STRING]"
-
-# You can also set the connection string for your normal development MS SQL database like below
-dotnet user-secrets set "Dapper:SqlServer" "[MSSQL_CONNECTION_STRING]"
-```
-
-You can then run just those tests from the `test/Infrastructure.IntegrationTest` folder using
-`dotnet test`.
+In your `server/dev/secrets.json` file find these two blocks of secrets:
+
+- This block is used for your primary Bitwarden database for each supported provider type. These are
+  what your running development server will connect to. You should already have passwords set up in
+  this block.
+  - If you do: ensure the username/password combos for each of your global database types are
+    applied to their siblings in the `databases` block.
+  - If you do not: please see the user secrets section of the server setup guide for doing this.
+  ```
+  "sqlServer": {
+    "connectionString": "Server=localhost;Database=vault_dev;User Id=SA;Password=___________;Encrypt=True;TrustServerCertificate=True;"
+  },
+  "postgreSql": {
+    "connectionString": "Host=localhost;Username=postgres;Password=_________;Database=vault_dev"
+  },
+  "mySql": {
+    "connectionString": "server=localhost;uid=root;pwd=_________;database=vault_dev"
+  },
+  "sqlite": {
+    "connectionString": "Data Source=/path/to/bitwardenServer/repository/server/dev/db/bitwarden.sqlite"
+  },
+  ```
```

Comment:
I don't think this is required. It's already covered in the sections above so this is largely repeating information. I think you could just start with the following paragraph and it would work just as well (or better because it's not repeating information).